appveyor.yml

```diff
@@ -0,0 +1,78 @@
+version: 0.0.0.0.1-branch-{branch}-build-{build}
+cache:
+    - c:\protobuf
+    - c:\protoc
+    - c:\zlib
+environment:
+    matrix:
+        - vc_arch: amd64
+          choco_arch:
+          nuget_arch: x64
+          target_arch: x86_64
+          qt_ver: 5.5\msvc2013_64
```

Comment:
For windows 5.5 is fine; 5.5 is problematic only for osx (rpath enabled by default, problems in detecting clang features).
Appveyor currently only supports qt 5.3, 5.4 and 5.5. We could download and install qt4 manually, but that would take a lot of time.